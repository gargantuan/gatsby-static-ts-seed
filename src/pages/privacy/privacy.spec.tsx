import * as React from 'react';
import * as renderer from "react-test-renderer"
import Privacy from './index';

describe("Index", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Privacy />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
