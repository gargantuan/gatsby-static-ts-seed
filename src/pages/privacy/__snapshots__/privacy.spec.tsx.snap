// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
Array [
  <h1>
    <a
      href="/"
    >
      Website Title
    </a>
  </h1>,
  <nav>
    <ul>
      <li>
        <a
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>,
  <h2>
    Privacy Policy
  </h2>,
  <h3>
    Edit this to be GDPR Compliant
  </h3>,
  <p>
    Your privacy is important to us. It is Your Company's policy to respect your privacy regarding any information we may collect from you across our website, 
    <a
      href="http://your-new-website.com"
    >
      http://your-new-website.com
    </a>
    , and other sites we own and operate.
  </p>,
  <p>
    We only ask for personal information when we truly need it to provide a service to you. We collect it by fair and lawful means, with your knowledge and consent. We also let you know why we’re collecting it and how it will be used.
  </p>,
  <p>
    We only retain collected information for as long as necessary to provide you with your requested service. What data we store, we’ll protect within commercially acceptable means to prevent loss and theft, as well as unauthorised access, disclosure, copying, use or modification.
  </p>,
  <p>
    We don’t share any personally identifying information publicly or with third-parties, except when required to by law.
  </p>,
  <p>
    Our website may link to external sites that are not operated by us. Please be aware that we have no control over the content and practices of these sites, and cannot accept responsibility or liability for their respective privacy policies.
  </p>,
  <p>
    You are free to refuse our request for your personal information, with the understanding that we may be unable to provide you with some of your desired services.
  </p>,
  <p>
    Your continued use of our website will be regarded as acceptance of our practices around privacy and personal information. If you have any questions about how we handle user data and personal information, feel free to contact us.
  </p>,
  <p>
    This policy is effective as of 9 May 2019.
  </p>,
  <p>
    Generated by 
    <a
      href="https://getterms.io/"
      title="Privacy Policy Template Generator"
    >
      GetTerms.io
    </a>
  </p>,
  <footer>
    <p>
      <a
        href="/privacy"
        title="privacy policy"
      >
        Privacy policy
      </a>
    </p>
  </footer>,
]
`;
